designer/client/conditions/inline-condition-model.js

```diff
@@ -21,6 +21,13 @@ export class ConditionsModel {
     return toReturn
   }
 
+  clear () {
+    this.#userGroupedConditions = []
+    this.#groupedConditions = []
+    this.#conditionName = undefined
+    return this
+  }
+
   name (name) {
```

Comment:
should this be a setter? (and getter). name looks like it just generates a name, but it's also doing some mutations. Feel free to leave as is, maybe just add a jsdoc?

Reply:
Fair point, I'll amend with my work on getting it wired in properly and persisted